_schemaV2.graphql

```diff
@@ -14392,6 +14398,10 @@ enum PartnerClassification {
   PRIVATE_DEALER
 }
 
+type PartnerCollectorProfile {
```

Comment:
This type doesn't seem to be doing anything currently, but I'm thinking that other metadata about the collector (like the `follows_profile` thing, some Exchange-backed summary data, and so on), will be added here.

In that case, cool! 👍 

However, `PartnerCollectorProfile` feels like a weird name for something that includes `collectorProfile: CollectorProfile` as a field. Maybe this type should be called `CollectorResume` or something of that sort? (should double-check the name). I think that would go a long way to help clearing up initial confusion as to why there are two types that have `-CollectorProfile` suffix.

Reply:
I wonder how we can transform data from [this presenter](https://github.com/artsy/gravity/blob/main/app/presenters/partner_collector_profile_presenter.rb#L1) to this type to avoid a double call to the resolver